### PR TITLE
Add create breakout room option to manage user menu

### LINKED ISSUE #6354: Move the "create breakout rooms action" to the manage users dropdown

### PR DESCRIPTION
**Pull request by KDSBrowne**
Adds an option in the user manage menu to create breakout rooms.
Combined with #6534 closes #6354

### CODE REVIEW

**Comment by Tainan404** [comment]
You forgot to remove the old code from actions bar.
peek 2019-01-14 10-34

**Comment by jfsiebel** [comment]
> You forgot to remove the old code from actions bar.
> peek 2019-01-14 10-34

https://github.com/bigbluebutton/bigbluebutton/issues/6354#issuecomment-447670157

**Comment by antobinary** [comment]
From #6354 
> In short, the final outcome is to display "Start breakout rooms" in both the "Manage Users" and "Actions" buttons.

@Tainan404 @jfsiebel looks like it's intended to be this way

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx [hunk 1]
```diff
@@ -187,8 +202,81 @@ class UserOptions extends PureComponent {
...
+        onClick={() => mountModal(<LockViewersContainer />)}
+      />),
+      (!meetingIsBreakout && !hasBreakoutRoom && isUserModerator
+        ? (
+          <DropdownListItem
+            key={this.createBreakoutId}
+            icon="rooms"
+            label={intl.formatMessage(intlMessages.createBreakoutRoom)}
+            description={intl.formatMessage(intlMessages.createBreakoutRoomDesc)}
+            onClick={() => this.handleCreateBreakoutRoomClick()}
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx]:
```suggestion
            onClick={this.handleCreateBreakoutRoomClick}
```

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx [hunk 2]
```diff
@@ -187,8 +202,81 @@ class UserOptions extends PureComponent {
...
+          />
+        ) : null
+      ),
+      (canInviteUsers
+        ? (
+          <DropdownListItem
+            icon="rooms"
+            label={intl.formatMessage(intlMessages.invitationItem)}
+            key={this.createBreakoutRoomId}
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx]: Looks like we need to make sure we do not have `canInviteUsers` and `!meetingIsBreakout && !hasBreakoutRoom && isUserModerator` both be true. The `DropdownListItem`s are the same in this case, including even the unique key

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx [hunk 3]
```diff
@@ -1,67 +1,47 @@
...
-    this.muteAllUsersExceptPresenter = this.muteAllUsersExceptPresenter.bind(this);
-    this.handleClearStatus = this.handleClearStatus.bind(this);
-  }
-
-  muteMeeting() {
-    const { muteAllUsers } = this.props;
-    muteAllUsers(Auth.userID);
-  }
-
-  muteAllUsersExceptPresenter() {
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/container.jsx]: Nice! I did not realize these were duplicated in user-list/service.js!

**Comment by antobinary** [comment]
Tested - works well. Once the uniqueness of elements in the dropdown is confirmed we can merge (user-list/user-list-content/user-participants/user-options/component.jsx )

**Pull request merged**